src/components/popover/widgets/capture-keystroke/capture-keystroke-button.component.ts

```diff
@@ -1,23 +1,76 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, EventEmitter, HostListener, Output } from '@angular/core';
+import { CaptureService } from '../../../../services/capture.service';
 
 @Component({
     selector: 'capture-keystroke-button',
     template: require('./capture-keystroke-button.component.html'),
     styles: [require('./capture-keystroke-button.component.scss')]
 })
-export class CaptureKeystrokeButtonComponent implements OnInit {
+export class CaptureKeystrokeButtonComponent {
+    @Output() capture = new EventEmitter<any>();
+
     private record: boolean;
+    private first: boolean; // enable usage of enter
```

Comment:
> The only minor issue I noticed is that when I trigger the button with Enter, the capture immediately stops and nothing gets captured. If it's easy to fix then you can do it, but it doesn't worth a lot of time.

This is for triggering capturing with enter when you are tabbing around the popover